webClient/src/app/app.component.css

```diff
@@ -58,6 +58,22 @@
   background-position: center;
 }
 
+.bookmark-button {
+  background-image: url(../assets/bookmark.png);
```

Comment:
This image has a white background, but the terminal has a grey background, so it looks weird.
Have you tried https://fontawesome.com/v4.7.0/icons/ ?
Zowe includes it, so you can use any of these icons. They're transparent so it is no problem.